framework/packages/abstract-interface/src/deployment.rs

```diff
@@ -93,6 +93,9 @@ impl<Chain: CwEnv> Deploy<Chain> for Abstract<Chain> {
             .version_control
             .register_natives(deployment.contracts())?;
 
+        // Approve abstract contracts if needed
+        deployment.version_control.approve_any_abstract_modules()?;
+
```

Comment:
Shouldn't we do it on a contract side?
Auto-accept if it's coming from Account(0) owner which has ABSTRACT_NAMESPACE namespace